cart.css

```diff
@@ -0,0 +1,78 @@
+@import "./style/header.css";
+@import "./style/footer.css";
+@import "./style/cards.css";
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+:root {
```

Comment:
the values here are mixed ,like you should either use color name or color code not both